frontend/src/Components/PostContentVideo.css

```diff
@@ -0,0 +1,13 @@
+.video-container {
+  position: relative;
+  width: 100%;
+  height: 0;
+  padding-bottom: 56.25%;
+}
+.youtube {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
```

Comment:
minior code style: I'd separate rules and add a new line at the end of the file to get rid of Github warning:
```css
.video-container {
  position: relative;
  width: 100%;
  height: 0;
  padding-bottom: 56.25%;
}

.youtube {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
}
```

Reply:
1)100% yes maybe it's make sense!  for big screens make a less width video frame. But reducing the width`(width:90%) `rule entails that the block with the video and the block with the `textarea  `will be unequal and unbalanced.  And I would like to video equal to the size of the field text area) 
2) Done
![Screenshot 2022-08-11 102527](https://user-images.githubusercontent.com/29427058/184084127-ee37bd0a-e3e9-4791-b2e7-f3d72e683016.jpg)
![Screenshot 2022-08-11 102601](https://user-images.githubusercontent.com/29427058/184084132-ec4008fc-6438-426d-8ea9-bc2faf6657c7.jpg)